tlzapp/app.go

```diff
@@ -244,7 +244,7 @@ func (a *App) serve(adminAddr string) error {
 
 	// set up HTTP client and request with short timeout and context cancellation
 	client := &http.Client{Timeout: 1 * time.Second}
-	req, err := http.NewRequestWithContext(ctx, http.MethodGet, "http://"+adminAddr, nil)
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, "http://localhost:12002", nil)
```

Comment:
This may not work if we listen on a very specific interface that isn't localhost, but this is already a problem on line [273](https://github.com/timelinize/timelinize/pull/26/files#diff-6e611fdd4dd4532ff7d2a10e9ce7caf886fbec2c836c4157ec8b73c43f705bd3L273), were localhost is also being used.

Reply:
Should we just use the TIMELINIZE_ADMIN_ADDR variable here too?

Reply:
Deets in https://github.com/timelinize/timelinize/pull/26#issuecomment-2299581913

Reply:
Thanks!